task/Update.js

```diff
@@ -14,19 +14,29 @@ const { router } = require('../config');
 const assert = require('assert');
 
 const MAX_GTFS_FALLBACK = 2; // threshold for aborting data loading
+const SPLIT_BUILD_TYPE = process.env.SPLIT_BUILD_TYPE || 'NO_SPLIT_BUILD';
```

Comment:
This is repeated in many places, maybe it should be it should be extracted to some util function so we ensure to have the same default everywhere.

Reply:
I added this to the config and imported it instead